apps/prairielearn/src/sprocs/instance_questions_calculate_stats.sql

```diff
@@ -9,6 +9,7 @@ WITH first_calculation AS (
         coalesce(bool_or(s.score = 1), FALSE)  AS some_perfect_submission_var,
         coalesce(bool_or(s.score != 0), FALSE) AS some_nonzero_submission_var,
         array_agg(s.score ORDER BY s.date)     AS submission_score_array_var,
+        array_agg(s.score ORDER BY s.date) FILTER (WHERE s.score IS NOT NULL) AS submission_non_null_score_array_var,
```

Comment:
As long as we're here, might as well get rid of the bespoke formatting on the other lines that tried to vertically align the aliases? Especially now that this line doesn't match the others.

Reply:
Done in 4c0fc0f7b73fec8205f6d5a89e43939fe0aea597.